File: C++/Maths/Area of Rectangle.cpp
#include <iostream>
using namespace std;

int main(){
    double length, width;
    cout << "Enter the length: ";
    cin >> length;
    cout << "Enter the width: ";
    cin >> width;

    double area = length * width;
    cout << "The area of rectangle is = " << area;
}

File: Java/Maths/Area of Rectangle.java
import java.util.Scanner;
class rectangle {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the length: ");
        double length = input.nextDouble();
        System.out.print("Enter the width: ");
        double width = input.nextDouble();

        double area = length * width;
        System.out.println("The area of rectangle is = " +  area);
    }
}